stdlib/public/SwiftShims/swift/shims/_SynchronizationShims.h

```diff
@@ -23,7 +23,7 @@
 #include <unistd.h>
 
 static inline __swift_uint32_t _swift_stdlib_gettid() {
-  static __thread tid = 0;
+  static __thread auto tid = 0;
```

Comment:
Id just use `__swift_uint32_t` because I don’t know if it will decide like `int` and just implicitly convert the return.

Reply:
Not that that’s a problem but rather just be explicit

Reply:
Makes sense to me. Updated!